src/components/forms/extra-question-form.js

```diff
@@ -262,6 +265,42 @@ class ExtraQuestionForm extends React.Component {
                         </div>
                     </div>
                 </div>
+                {entity.usage === 'Ticket' && 
```

Comment:
remove this condition , ticket_type_ddl.length >0 and badge_features_ddl.length  should be ebough